vllm/utils.py

```diff
@@ -36,3 +37,6 @@ def get_cpu_memory() -> int:
 
 def random_uuid() -> str:
     return str(uuid.uuid4().hex)
+
+def in_wsl() -> bool:
+    return "microsoft" in " ".join(uname()).lower()
```

Comment:
A quick question: how reliable is it? If it is also used in other repo, could you provide a reference?